### DISCUSSION TITLE
How to rapidly get the values of a list of variables?

### DISCUSSION THREAD

**Discussion by styw1234** [yalmip/YALMIP#1405] [General discussion]
Usually, we can use 'value' to get the value of the variable.
For example:

a=sdpvar(1,1)
value_a=value(a)

Suppose that there are many variables which are generated by 'sdpvar'.
How to rapidly get the values of these variables ?
Is there any tip or function to get the value of a list of variables?

**Comment by johanlofberg** [comment]
Things like value(recover(myconstraints)) or value(recover(whatever)), or value(recover(1:yalmip('nvars'))). Problem is of course that you have no idea which variable you just computed the value of (unless you understand by the ordering)

**Reply by styw1234** [reply]
Thank you.

Based on your advice, I write some codes to change the variables of 'sdpvar' to value. It is a success!

https://www.mathworks.com/matlabcentral/fileexchange/165981-sdpvartovalue
